### PR TITLE
Hotfix for updates button causing execution exception

### PR DESCRIPTION
**Pull request by ntreese**
There was a bug that caused an execution exception when pressing the updates button in the admin panel. It was caused by a StringIndexOutOfBounds error.

### CODE REVIEW

**Review by ntreese — COMMENTED** [review]
The code looks good to me. I really like the documentation on the internet test script.

**Comment by nbaggett** [comment]
The ping is hard coded because the new framework doesn't have a latency test, so we need to eventually write our own. Andy recommended that we leave this for next year, so I added some documentation and mentioned that in it.

File: app/femr/business/services/system/UpdatesService.java [hunk 1]
```diff
@@ -75,9 +84,52 @@ public UpdatesService(IRepository<INetworkStatus> networkStatusRepository,
...
         try {
-            data = BackEndControllerHelper.executeSpeedTestScript("speedtest/sptest.py");
-            //Update Status
-            Float Ping = Float.parseFloat(data.get(2));
+            // DOWNLOAD SPEED TEST SOCKET
+            SpeedTestSocket speedTestSocketDownload = new SpeedTestSocket();
+            speedTestSocketDownload.addSpeedTestListener(new ISpeedTestListener() {
+                @Override
+                public void onCompletion(SpeedTestReport report) {
+                    System.out.println("DOWNLOAD [COMPLETED] rate in bit/s   : " + report.getTransferRateBit());
```
Comment (42cleslie) [inline app/femr/business/services/system/UpdatesService.java:92]: do we want these print statements?

**Review by 42cleslie — APPROVED** [review]
Not sure if we want to be printing stuff out. Maybe we do. Other than that I'd say good to merge.

**Comment by 42cleslie** [comment]
It also says image failed to build inn Docker Hub. Not sure if that's a big problem

**Review by oshorab — APPROVED** [review]

**Pull request merged**